templates/prusa-mk3s-printer.template.cfg

```diff
@@ -0,0 +1,229 @@
+
+#############################################################################################################
+### MACRO CONFIGURATION
+### Configure the behavior of RatOS macros
+#############################################################################################################
+[gcode_macro RatOS]
+# Use absolute extrusion mode
+# Set to True to use relative extrusion mode
+variable_relative_extrusion: False
+# Wait for extruder to reach 150 so an inductive probe (if present) is at a predictable temp. 
+# Also allows the bed heat to spread a little, and softens any plastic that might be stuck to the nozzle.
+# Set to False to disable
+variable_preheat_extruder: True
+# Calibrate the bed mesh in the START_PRINT macro.
+# Set to false to skip BED_MESH_CALIBRATE, it will still load the BED_MESH 
+# with the name "ratos", be sure to save your bed_mesh profile with that name.
+# or override the _START_PRINT_BED_MESH macro to implement your own mesh handling logic.
+variable_calibrate_bed_mesh: True
+# Print a prime line at the end of the START_PRINT macro
+# set to False to disable nozzle_priming.
+variable_nozzle_priming: "primeline"
+# Park in the back when waiting for the extruder to heat up
+# set to "front" to park in the front, or "center" to park in the center.
+variable_start_print_park_in: "front"
+# Height to park it when waiting for extruder to heat.
+variable_start_print_park_z_height: 50
+# Skew profile to load before starting the print 
+# uncomment this to use your calibrated skew correction profile.
+variable_skew_profile: "my_skew_profile"
+# Park in the back after the print has ended or was cancelled.
+# set to "front" to park in the front, or "center" to park in the center.
+variable_end_print_park_in: "back"
+# Park in the back when the print is paused.
+# set to "front" to park in the front, or "center" to park in the center.
+variable_pause_print_park_in: "front"
+# Set the speed for travel moves in RatOS Macros in mm/s.
+variable_macro_travel_speed: 200
+# Set the length and speed for the unload filament
+variable_filament_unload_length: 100
+variable_filament_unload_speed: 5
+# Set the length and speed for the load filament
+variable_filament_load_length: 100
+variable_filament_load_speed: 10
+
+#############################################################################################################
+### PRINTER CONFIGURATION
+### Customize the defaults to your specific build
+#############################################################################################################
+
+[mcu]
+serial: /dev/prusa-einsy
+
+[tmc2130 extruder]
+sense_resistor: 0.22
+
+[extruder]
+pressure_advance: 0.04 #this is a reference value, should be calibrated on each printer
+control = pid
+pid_Kp: 16.13
+pid_Ki: 1.1625
+pid_Kd: 56.23
+
+[heater_bed]
+control = pid
+pid_Kp: 126.13
+pid_Ki: 4.3
+pid_Kd: 924.76
+
+[pause_resume]
+recover_velocity: 50.
+
+[stepper_z]
+position_max: 214
+
+[probe]
+z_offset = 2.0
```

Comment:
This should be 0 to avoid accidents.

Reply:
Yes, you are right I noticed it in my installation